README.md

```diff
@@ -1,20 +1,64 @@
-# my-component
+# Snapshot (SCD) SQL generator
 
-[![Build Status](https://travis-ci.com/keboola/my-component.svg?branch=master)](https://travis-ci.com/keboola/my-component)
+[![Build Status](https://travis-ci.com/keboola/app-transformation-pattern-scd.svg?branch=master)](https://travis-ci.com/keboola/app-transformation-pattern-scd)
 
-> Fill in description
+> Generate code for SCD type 2 or 4 on any input dataset.
 
-# Usage
+## Configuration Options
 
-> fill in usage instructions
+- scd_type: enum {`scd2`, `scd4`} (required)
+- primary_key: string (required)
+- monitored_parameters: string (optional)
+- deleted_flag: boolean (optional) - default value is `false`
+- use_datetime: boolean (optional) - default value is `false`
+- keep_del_active: boolean (optional) - default value is `false`
+- timezone: string (required)
+
+## SCD type
+Type of the snapshot generated snapshot / slowly changing dimension.
```

Comment:
Tejto vete nejak nerozumiem, co sa tym chce povedat?

Reply:
Mozno `Type of the slowly changing dimension.`?

Reply:
mělo tam být `Type of the generated snapshot / slowly changing dimension.` ale co píšeš ty je taky v pohodě nebo by ještě jednodušeji mohlo být  `Type of the generated slowly changing dimension` a možná by šlo i to jak píšeš ty

Reply:
upravil jsem to jak si psal ty....